inc/http-client/class-cache-strategy.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace RemoteDataBlocks\HttpClient;
+
+use Kevinrob\GuzzleCache\Strategy\GreedyCacheStrategy;
+use Psr\Http\Message\RequestInterface;
+use RemoteDataBlocks\Logging\LoggerManager;
+
+class CacheStrategy extends GreedyCacheStrategy {
```

Comment:
Worth s/`CacheStrategy`/`CustomizedCacheStrategy`?

Just thinking, in other context, like when instantiated, some sort of qualifier like that might convey more meaning. And `RemoteDataBlocksCacheStrategy` feels a tad long even for me (though I'm open to it!).

Reply:
I ended up prefixing these with `Rdb` -- feel free to push a change if anyone has a preference.